src/Functions/mortonEncode.cpp

```diff
@@ -321,6 +321,9 @@ class FunctionMortonEncode: public TargetSpecific::Default::FunctionMortonEncode
 
     ColumnPtr executeImpl(const ColumnsWithTypeAndName & arguments, const DataTypePtr & result_type, size_t input_rows_count) const override
     {
+        if (input_rows_count == 0)
```

Comment:
Ok.

I see a way to automate it.

Shouldn't all functions return something like `return_type->createColumn()` for empty blocks?

If there are special cases, we can make `useDefaultImplementationForEmptyBlocks` :)

Reply:
> Shouldn't all functions return something like return_type->createColumn() for empty blocks?

Most likely.
I'll add it to `IExecutableFunction::execute` and see what happens.